doc/tutorials/gapi/face_beautification/face_beautification.markdown

```diff
@@ -1,5 +1,7 @@
 # Implementing a face beautification algorithm with G-API {#tutorial_gapi_face_beautification}
 
+@prev_tutorial{tutorial_gapi_anisotropic_segmentation}
+
 [TOC]
```

Comment:
BTW, Also there is another way to specify table of contents. [Page](http://pullrequest.opencv.org/buildbot/export/pr/18712/docs/d4/d48/tutorial_gapi_face_beautification.html)

But I agree that we need to check/fix headers on problematic pages.